react/features/base/jwt/middleware.js

```diff
@@ -183,11 +282,24 @@ function _setJWT(store, next, action) {
             action.issuer = iss;
             if (context) {
                 action.callee = context.callee;
-                action.caller = context.user;
+                action.caller = _normalizeCallerFields(context.user);
```

Comment:
I talked to @paweldomas (who is not the author of the "problem") in person about it but I'd like to include it here for @saghul and whoever else may agree: using multiple names for one and the same abstraction is plain and needlessly confusing! With the overwriting that we do now in this PR, we get to a total of 3 names: `user`, `caller`, and `localParticipant`. So in the "overwrite" and "undo overwrite" functions I was seeing `caller` and `localParticipant`, I was reading the comments, and I thought `caller` was inside/a property of `context.user`...